packages/contracts/src/utils/asset_proxy_utils.ts

```diff
@@ -4,37 +4,35 @@ import ethUtil = require('ethereumjs-util');
 
 import { AssetProxyId } from './types';
 
-export function encodeAssetProxyId(assetProxyId: AssetProxyId): Buffer {
-    return ethUtil.toBuffer(assetProxyId);
-}
-
-export function encodeAddress(address: string): Buffer {
-    if (!ethUtil.isValidAddress(address)) {
-        throw new Error(`Invalid Address: ${address}`);
-    }
-    const encodedAddress = ethUtil.toBuffer(address);
-    return encodedAddress;
-}
-
-export function encodeUint256(value: BigNumber): Buffer {
-    const formattedValue = new BN(value.toString(10));
-    const encodedValue = ethUtil.toBuffer(formattedValue);
-    return encodedValue;
-}
-
-export function encodeERC20ProxyData(tokenAddress: string): string {
-    const encodedAssetProxyId = encodeAssetProxyId(AssetProxyId.ERC20);
-    const encodedAddress = encodeAddress(tokenAddress);
-    const encodedMetadata = Buffer.concat([encodedAssetProxyId, encodedAddress]);
-    const encodedMetadataHex = ethUtil.bufferToHex(encodedMetadata);
-    return encodedMetadataHex;
-}
-
-export function encodeERC721ProxyData(tokenAddress: string, tokenId: BigNumber): string {
-    const encodedAssetProxyId = encodeAssetProxyId(AssetProxyId.ERC721);
-    const encodedAddress = encodeAddress(tokenAddress);
-    const encodedTokenId = encodeUint256(tokenId);
-    const encodedMetadata = Buffer.concat([encodedAssetProxyId, encodedAddress, encodedTokenId]);
-    const encodedMetadataHex = ethUtil.bufferToHex(encodedMetadata);
-    return encodedMetadataHex;
-}
+export const proxyUtils = {
```

Comment:
Let's keep names consistent between file and module name. This module should be named `assetProxyUtils`.